cohttp-lwt/src/body.ml

```diff
@@ -75,7 +75,7 @@ let drain_body (body:t) =
 
 let of_string_list l = `Strings l
 
-let of_stream s = `Stream s
+let of_stream s = `Stream (Lwt_stream.filter (fun s -> s <> "") s)
```

Comment:
Can this, or the equivalent Pipe version, hang due to a broken infinite stream of empty lists?

Reply:
Surely no more than it would hang if the first element in the pipe is indefinitely delayed, I would think.